fastlane/Fastfile

```diff
@@ -11,7 +11,6 @@ APP_STORE_METADATA_FOLDER = File.join(FASTLANE_FOLDER, 'metadata')
 SECRETS_FOLDER = File.join(Dir.home, '.configure', 'pocketcasts-ios', 'secrets')
 ASC_KEY_PATH = File.join(SECRETS_FOLDER, 'app_store_connect_fastlane_api_key.json')
 VERSION_XCCONFIG_PATH = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.xcconfig')
-SPM_CACHE_FOLDER = File.join(PROJECT_ROOT_FOLDER, 'vendor', 'spm')
```

Comment:
The CI Toolkit action restores the cache to `~/Library/Caches/org.swift.swiftpm` (where Xcode already checks for packages), so we don't have to specify a cache location in the Fastfile.